geomesa-gs-plugin/geomesa-bigtable-gs-plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>geomesa-gs-plugin</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <version>1.2.7-SNAPSHOT</version>
```

Comment:
Arent' we using ```${project.version}``` elsewhere, rather than hard-coding this, or is this a special case?